app/uk/gov/hmrc/securemessage/controllers/utils/MessageSchemaValidator.scala

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage.controllers.utils
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.networknt.schema.{ JsonSchema, JsonSchemaFactory, SpecVersion, ValidationMessage }
+import play.api.libs.json.JsValue
+
+import scala.collection.JavaConverters._
+
+trait MessageSchemaValidator {
+
+  lazy val messageSchema: String =
```

Comment:
Could this go in a separate file (txt file) or something?

Reply:
@PaulThor That is what I tried, keeping json in a separate json file. Somehow, the file is reading during deployment process, pr-builder and release jobs are fine. I tried different options, like keeping the file in different folder and used different API for file load, nothing worked out. So, thought of doing it later,